src/components/searchBar/SearchBar.jsx

```diff
@@ -1,11 +1,7 @@
-import React from 'react'
+import React from 'react';
 
 function SearchBar() {
-    return (
-        <div>
-            
-        </div>
-    )
+  return <div></div>;
 }
 
-export default SearchBar
+export default SearchBar;
```

Comment:
Will this component be reused? Maybe _CitySearchBar_ name will be better.

Reply:
this comonent can be used in different projects that is why i decided to put it inside components

Reply:
Ok. Why did you name internal components like _CitiesSearch_ and _SearchedCity_? Maybe these component names would be without _City_ and _Cities_ ?